migration/README.md

```diff
@@ -0,0 +1,19 @@
+# unified-codes : data migration
+
+A dump of the v1 data is in `data/items.json`.
+In the root is a node script which will parse and transform to a dgraph format, ready for input.
+
+## Usage
+
+Run the migration script `node migrate.js [source] [output]`
+e.g. `node migrate.js ./data/items.json ./data/data.v2.json`
+
+
+### Add/update data
+
+You can use the project in the `dgraph` folder to clean and import the results
+```
+../dgraph/clean.sh
+../dgraph/alter.sh ../dgraph/sample/schema.gql 
+../dgraph/mutate.sh ./data/data.v2.json
```

Comment:
I'm sorry - I think I may have deleted mutate.sh with the Dgraph GraphQL changes! (because adding data needs to be done through GraphQL layer for it to be accessible via that layer - and mutating and querying use the same endpoint in Dgraph GraphQL). Feel free to add it back in until we have a better solution (seeing all these JSON files is making it seem more worthwhile to write a script to copy JSON body into GraphQL mutation request).

Reply:
all good - I'll update after moving the repo :)